tidyconfig.txt

```diff
@@ -0,0 +1,4 @@
+char-encoding: utf8
+indent: yes
```

Comment:
FWIW, we use auto in other specs. E.g. https://github.com/w3c/preload/blob/gh-pages/tidyconfig.txt